docs/RestClients.md

```diff
@@ -52,13 +51,13 @@ You can find REST clients for various backends in third-party repositories:
 
 If you've written a REST client for another backend, and open-sourced it, please help complete this list with your package.
 
-### Simple REST
+### Usage
 
 This REST client fits APIs using simple GET parameters for filters and sorting. This is the dialect used for instance in [FakeRest](https://github.com/marmelab/FakeRest).
 
 | REST verb            | API calls
```

Comment:
In the `next` branch, this table comes *after* the React code to setup the rest client. Please follow the same plan.